src/docs/components/tools/vendorCookieInspector/vendorCookieInspector.jsx

```diff
@@ -123,13 +123,37 @@ export default class VendorCookieDecoder extends Component {
 		this.setState({ localCookie: readCookie(VENDOR_CONSENT_COOKIE_NAME) });
 	};
 
+	readParams = () => {
+		var vendor_cookie = this.getUrlParam("string");
+		if (typeof(vendor_cookie) === "string") {
+			this.setState({ b64: vendor_cookie});
+		}
+	}
+
+	getUrlVars = () => {
+		var vars = {};
+    var parts = window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function(m,key,value) {
```

Comment:
You should make this an actual query param, so you can just parse `window.location.search` and find your value easier.  Your URL would be in the form `/docs/?base64-value={value}#/tools/vendor-cookie-inspector`